users/jackcasstlesjones/new-personal-website/index.html

```diff
@@ -21,14 +27,46 @@
   </head>
   <body>
     <header>
+      <?xml version="1.0" encoding="UTF-8"?><svg
+        class="mobile-menu-icon"
+        width="24px"
+        height="24px"
+        stroke-width="1.5"
+        viewBox="0 0 24 24"
+        fill="none"
+        xmlns="http://www.w3.org/2000/svg"
+        <!--
+        color="#f4edea"
+        --
+      >
+        >
+        <path
+          d="M3 5H21"
+          stroke-width="2"
+          stroke-linecap="round"
+          stroke-linejoin="round"
+        ></path>
+        <path
+          d="M3 12H21"
+          stroke-width="2"
+          stroke-linecap="round"
+          stroke-linejoin="round"
+        ></path>
+        <path
+          d="M3 19H21"
+          stroke-width="2"
+          stroke-linecap="round"
+          stroke-linejoin="round"
+        ></path>
+      </svg>
       <nav id="home" class="navbar">
         <div class="navbar-img-and-title">
           <img class="navbar-img" src="assets/portrait.png" alt="" />
           <h1>Jack Casstles-Jones</h1>
         </div>
         <ul class="navbar-links-list">
           <li>
-            <a id="navbar-home" class="navbar-link" href="#home">Home</a>
+            <a id="navbar-home" class="navbar-link" href="#">Home</a>
           </li>
           <li>
             <a id="navbar-about" class="navbar-link" href="#about">About</a>
```

Comment:
You can update the navbar-photography href to "#photography" now the section is added